### DISCUSSION TITLE
Amiral Battı

### DISCUSSION THREAD

**Discussion by kaanpinar04** [sonsuzus/sonsuzus#501] [Soru Cevap]
Arkady amiral battı oynuyor.Bu oyunun kuralları pek önemli değil.

NxN bölmeli bir alan var. Bu alanda bir adet k-decker(k uzunluğunda yatay veya dikey hizalanmış gemi) olmalı. Fakat Arkady bu geminin konumunu bilmiyor. Arkady her bölmenin kesinlikle boş mu olduğunu yoksa geminin parçasını mmı bulundurduğunu biliyor.

Geminin bütün potansiyel konularını göz önünde bulundurun.

Girdi
İlk satır n ve k tam sayıları(1<=k<=n<=100-k alanın boyutu, n geminin boyutu) bulundurur.

Bir sonraki n satır alanı bulundurur. Her satır n kadar karakter bulundurur, her biri ya #(kesinlikle boş bölme) ya da .(geminin bulunabileceği bölme) bulundurur.

Çıktı

İki tam sayıyı çıktı olarak verir(gemi bulunma ihtimali en yüksek olan bölmenin satır ve sütun numarası).

Eğer birden fazla cevap varsa herhangi birini çıktı olarak verir Eğer geminin olabileceği hiç bir bölme yoksa herhangi bir bölmeyi çıktı olarak verir

**Comment by FurkanYildirim1350** [comment]
```
def eniyi(n,k,alan):
    skor=-1
    eniyipozisyon=(1,1)
    for i in range(n):
        for a in range(n-k+1):
            uyumlu=True
            for e in range(k):
                if alan[i][a+e]!='.': uyumlu=False;break
            if uyumlu:return(i+1,a+1)
    for a in range(n):
        for i in range(n-k+1):
            uyumlu=True
            for e in range(k):
                if alan[i+e][a]!='.': uyumlu=False;break
            if uyumlu:return(i+1,a+1)
    return eniyipozisyon
n,k=map(int,input("alan ve gemi:").split())
alan=[input("kuşbakışı alan:\n")for t in range(n)]
sonuç=eniyi(n,k,alan)
print(sonuç[0],sonuç[1])
```
